__tests__/CanCommunity.test.ts

```diff
@@ -961,7 +979,13 @@ describe('test CanCommunity', () => {
       expect(serializeActionData).toHaveBeenNthCalledWith(3, _options, ActionNameEnum.SETVOTER, setVoterInput);
       expect(serializeActionData).toHaveBeenNthCalledWith(4, _options, ActionNameEnum.SETVOTERULE, setVoteRuleInput);
       expect(serializeActionData).toHaveBeenNthCalledWith(5, _options, ActionNameEnum.SETEXECTYPE, setExecTypeInput);
-      expect(execCode).toBeCalledWith(CODE_IDS.SET_RIGHT_HOLDER_FOR_CODE, codeActions, CodeTypeEnum.AMENDMENT, undefined);
+      expect(execCode).toBeCalledWith(
+        CODE_IDS.SET_RIGHT_HOLDER_FOR_CODE,
+        codeActions,
+        CodeTypeEnum.AMENDMENT,
+        undefined,
+        input.code_id,
+      );
     });
```

Comment:
i found 11 results using executeCode, so plz update input of all of them @danielAlvess 

![Screen Shot 2020-03-18 at 3 50 55 PM](https://user-images.githubusercontent.com/7411225/76942480-64e07600-6930-11ea-91ac-a403e2af1b52.png)